src/plugins/intel_gna/src/gna_plugin_config.cpp

```diff
@@ -46,6 +46,39 @@ static const caseless_unordered_map<std::string, std::pair<Gna2AccelerationMode,
 };
 OPENVINO_SUPPRESS_DEPRECATED_END
 
+/**
+ * @brief Parse string into value without throwing
+ * @param value A parsed value
+ * @param def_value A value to return in case of exception
+ * @return parsed value, default value otherwise
+ * @ingroup ov_runtime_gna_prop_cpp_api
+ */
+template <class T>
+static T FromString(const std::string& value, T def_value) {
```

Comment:
You could also try to mapping enum to string:
https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gna/src/common/misc_utils.hpp#L23

example here
https://github.com/openvinotoolkit/openvino/blob/b70e56d11039c9d3fbb774f2bee4c92304f11fe9/src/plugins/intel_gna/src/common/gna_target.cpp#L82

Reply:
We can think about changing the way of handling properties, but it would require deeper refactoring. 
Leaving it for now.